package.json

```diff
@@ -124,6 +124,7 @@
   },
   "devDependencies": {
     "lerna": "^6.2.0",
+    "source-map-loader": "4.0.1",
```

Comment:
Should this be added to `builder/package.json` instead of the top-level `package.json`?

Reply:
Thanks for that suggestion. Let me try that.

Reply:
Right since the changes above are in the `builder` it looks like the dependency should be added to that package.